src/ServerHandler.cpp

```diff
@@ -15,7 +15,11 @@ ServerHandler::~ServerHandler()
 	}
 
 	if (!_servers.empty())
+	{
+		for (auto serv : _servers)
```

Comment:
you're using auto. my head just exploded.

Reply:
I know Im getting tired of carrying this group

Reply:
😂